proxmox/config_qemu.go

```diff
@@ -1175,8 +1175,17 @@ func FormatDiskParam(disk QemuDevice) string {
 		diskConfParam = append(diskConfParam, diskMountOpts)
 	}
 
+	// Backup
+	if backup, ok := disk["backup"].(bool); ok {
+		// Backups are enabled by default (backup=1)
+		// Only set the parameter if backups are explicitly disabled
+		if backup == false {
```

Comment:
can you please make the audit happy @tnyeanderson 🙏

Reply:
done. can you rerun the workflow?